Probesys/bileto
# imp: Show an icon on the messages sent by email

## Related issue(s)



https://github.com/Probesys/bileto/issues/43

## Changes



- add an envelope icon
- show the icon on the message sent by email
- add `email` as a valid value of Message vias

## How to test manually



- send an email to support@
- go to the created ticket
- check the message has an icon (check an explanation appears on hover)

## Checklist
